Mailing.rb

```diff
@@ -0,0 +1,140 @@
+require 'net/smtp'
+require 'pp'
+require 'active_record'
+require 'time'
+require 'time_difference'
+@config = YAML.load_file('config.yml')
+
+# def send_time_check user
+#   utc_time = Time.now.getutc
+# end
+
+def mail_send user_to
+
+  repo = @config['repositories'][0]['name']
+
+  pull_requests = PullRequest.all.where(state: 'open')
+
+  other_block = []
+  your_block = []
+
+  pull_requests.each do |pull_request|
+
+    start_time = pull_request.added_to_database
+    Time.parse(start_time)
+    end_time = Time.now
+    conflict_time = TimeDifference.between(start_time, end_time).in_hours.to_i
+    conflict = "#{conflict_time} hours"
+
+    mergeable = pull_request.mergeable
+    if mergeable
+      merg_status = "<span style='color:green;'><b>No</b></span>"
+    else
+      merg_status = "<span style='color:red;'><b>Yes</b></span> <b>#{conflict}</b>"
+    end
+
+    merg_state = ''
+    importance = 3
+    case pull_request.mergeable_state
+      when 'clean'
+        merg_state = "<span style='color:green;'><b>Stable</b></span>"
+      when 'unstable'
+        importance = 2
+        merg_state = "<span style='color:red;'><b>Unstable</b></span> <b>#{conflict}</b>"
+      when 'dirty'
+        importance = 1
+        merg_state = "<span style='color:red;'><b>Unstable</b></span> <b>#{conflict}</b>"
+      else
+    end
+
+    block = ({index: importance, text: "
```

Comment:
Make method that will have template and accepts all template variables as parameter.